addons/mrp_subcontracting/models/stock_move_line.py

```diff
@@ -1,20 +1,21 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import api, models
 
 
 class StockMoveLine(models.Model):
     _inherit = 'stock.move.line'
 
-    def create(self, values):
-        records = super(StockMoveLine, self).create(values)
-        records.filtered(lambda ml: ml.move_id.is_subcontract).move_id._check_overprocessed_subcontract_qty()
+    @api.model_create_multi
+    def create(self, vals_list):
+        records = super(StockMoveLine, self).create(vals_list)
+        records.move_id.filtered("is_subcontract")._check_overprocessed_subcontract_qty()
         return records
 
     def write(self, values):
         res = super(StockMoveLine, self).write(values)
-        self.filtered(lambda ml: ml.move_id.is_subcontract).move_id._check_overprocessed_subcontract_qty()
+        self.move_id.filtered("is_subcontract")._check_overprocessed_subcontract_qty()
```

Comment:
Is this part bringing a performance benefit? I would do code optimisation is master instead of stable.

Reply:
As everything is in cache anyway, it should only be a minor improvement, indeed, i'll remove it (I did a branch for master in the beginning, but thought it was worth cherry-picking some main models create_multi fixes for v13, but I forgot to remove this part).